test/spec/svg/mvt.js

```diff
@@ -0,0 +1,98 @@
+describe('iD.svgMvt', function () {
+    var context;
+    var surface;
+    var dispatch = d3.dispatch('change');
+    var projection = iD.geoRawMercator()
+        .translate([6934098.868981334, 4092682.5519805425])
+        .scale(iD.geoZoomToScale(17))
+        .clipExtent([[0, 0], [1000, 1000]]);
+
+
+    var gj = {
+        'type': 'FeatureCollection',
+        'features': [
+            {
+                'type': 'Feature',
+                'id': 316973311,
+                'geometry': {
+                    'type': 'Point',
+                    'coordinates': [
+                        -74.38928604125977,
+                        40.150275473401365
+                    ]
+                },
+                'properties': {
+                    'abbr': 'N.J.',
+                    'area': 19717.8,
+                    'name': 'New Jersey',
+                    'name_en': 'New Jersey',
+                    'osm_id': 316973311
+                }
+            }
+        ]
+    };
+
+    beforeEach(function () {
+        context = iD.coreContext();
+        d3.select(document.createElement('div'))
+            .attr('id', 'map')
+            .call(context.map().centerZoom([-74.389286, 40.1502754], 17));
+
+        surface = context.surface();
+    });
+
+    it('creates layer-mvt', function () {
+        var render = iD.svgMvt(projection, context, dispatch);
+        surface.call(render);
+
+        var layers = surface.selectAll('g.layer-mvt').nodes();
+        expect(layers.length).to.eql(1);
+    });
+
+    it('draws geojson', function () {
+        var render = iD.svgMvt(projection, context, dispatch).geojson(gj);
+        surface.call(render);
+
+        var path = surface.selectAll('path.mvt');
+        expect(path.nodes().length).to.eql(1);
+        expect(path.attr('d')).to.match(/^M.*z$/);
+    });
+
+    describe('#url', function() {
+        it('handles pbf url', function () {
+            var url = 'https://api.mapbox.com/v4/mapbox.mapbox-streets-v6/9/150/194.vector.pbf?access_token='
```

Comment:
We should avoid linking to a mapbox streets pbf here.
Can you try to use [sinon](http://sinonjs.org/releases/v6.1.3/
) to fake a response?

Here is an example in the OpenStreetCam test:
https://github.com/openstreetmap/iD/blob/master/test/spec/services/openstreetcam.js#L54-L105

I think it would be ok to store .pbf test fixture in the `test/data` folder, and use [`fs.readFile`](https://nodejs.org/dist/latest-v6.x/docs/api/fs.html) to load the data .